HouseRules_Essentials/Rulesets/AoePotionsAndBuffsRuleset.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace HouseRules.Essentials.Rulesets
+{
+    using System.Collections.Generic;
+    using DataKeys;
+    using HouseRules.Essentials.Rules;
+    using HouseRules.Types;
+
+    internal static class AoePotionsAndBuffsRuleset
```

Comment:
Note: Named as such because class names can not begin with a number (e.g., `3x3Potions`).  Feel free to rename to anything better.